src/renderer/utils/cachedLastUsedPath.ts

```diff
@@ -0,0 +1,14 @@
+export function cachedLastUsedPath(
```

Comment:
Is this really "caching"? You're _persisting_ the last used directory in a storage. I'd call this method `storeLastUsedPath` or similar.

Reply:
it's only stored for the current session though